### ISSUE TITLE
printing MD5 in welcome-email.phtml 

### ISSUE THREAD

**Issue by bcix** [inex/IXP-Manager#202]
Hi colleagues,

I'm just working on a modified template for the welcome email based on the great INEX template in
application/views/_skins/inex/customer/email/welcome-email.phtml.

I want to print out the IPv4 and IPv6 MD5 in the welcome email as well. This works, but only for customers with **one** connection. For customers with more than one connection it remain empty.

I just added it like this in line 4:
```
{if $vi->getIpv6enabled()}
IPv6 Address:  {$vi->getIPv6Address()->getAddress()}/{$netinfo.$vlanid.6.masklen}
IPv6 Hostname: {$vi->getIpv6hostname()}
IPv6 MD5 for route server: {$vi->getIpv6bgpmd5secret()}
{else}
IPv6:          Please contact us at {$options.identity.email} to enable IPv6
{/if}
```

Any suggestion to make it work for customers with more than one connection?

Thanks, Thorleif

**Comment by barryo** [comment]
@bcix - hi Thorleif.

This is actually a bug in our template which I corrected in the above commit. It's a very hacky-INEX specific fix (but hey, it's our own skin!) as the schema doesn't have proper relational objects for route servers and collectors but this is something we're working on.

Let me know if you need more help than the commit gives.

 - Barry

**Comment by bcix** [comment]
Hi Barry,

sorry, this new version also do not work for peers with two (or more ) connections per peering LAN.
And the last thing "per peering LAN" was a misunderstanding between us. We only have one peering LAN, but more and more customers with more than one connection in this one peering LAN. 
I'll specify this in future issues in more detail.

For testing I copied the new file 1:1 in _skins/bcix/customer/email , modified the VLAN ID to 100 (BCIX Peering LAN ) and do not see any output at line 217, if a customer has more than one connection.
For a customer with only one connection , it works as expected. 

From my understanding in  ...foreach $connection..  all MD5s has to placed in variables, which can
be printed out, later. I already tried this, but I'm not "firm" enough with the template 
language and unsuccessful :-( 

Thanks, Thorleif
